subprojects/docs/src/snippets/buildCache/cacheable-bundle/kotlin/build.gradle.kts

```diff
@@ -39,7 +45,9 @@ tasks.register<NpmTask>("bundle") {
 
 tasks.register("printBundle") {
     dependsOn("bundle")
+
+    val projectLayout = layout
     doLast {
-        println(file("$buildDir/bundle.js").readText())
+        println(layout.buildDirectory.file("bundle.js").get().asFile.readText())
```

Comment:
You meant to use `projectLayout` here, right?

This causes this sample to fail verification. You probably missed it because you tried to verify your fix after removing the snippets from the list of failing tests, but kept using `-PrunBrokenConfigurationCacheDocsTests=true`. I reckon this might be surprising, but if you run the tests with that option set to `true`, only tests in the list from broken tests will be run, any other tests will be skipped (leading to a successful build result, if you removed the failing tests before they were fixed from the list).

Reply:
BTW, please beware I force-pushed to your branch to perform a sync with master.

Reply:
Updated hackathon's workflow and faq documents to clarify that: https://github.com/gradle/cc-hackathon-2022/pull/6